src/components/ModalWindow/ModalWindow.module.scss

```diff
@@ -3,9 +3,14 @@
 }
 
 input:invalid {
-  border: 1px solig rgb(255, 0, 0) !important;
+  border: 1px solig rgb(211, 3, 3) !important;
 }
 
 input {
   border: 1px solig rgb(0, 0, 0) !important;
+}
+
+.formControl {
+  background-color: #ff000021 !important;
```

Comment:
I don't like !important

Reply:
In this case it was necessary to use it, I also try not to use it without needs